test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
+        }
+        return HTMLElement.call(this) || this;
```

Comment:
Although I think we are assuming that where `Reflect` is not present, you must have the polyfill + shim...